README.md

```diff
@@ -97,15 +97,15 @@ Python: [PyQT5](https://pypi.org/project/PyQt5/)
 
 ```bash
 sudo pacman -S clang cmake bpf libbpf libnetfilter_queue spdlog boost libmnl \
-    nlohmann-json python3 python-pyqt5 
+    nlohmann-json conntrack-tools python3 python-pyqt5 
 ```
 
 ### Installing dependencies on Ubuntu 20.10 (minimum version)
 
 ```bash
 sudo apt-get install cmake clang libboost-all-dev libspdlog-dev \
     libnetfilter-queue-dev libmnl-dev linux-tools-common nlohmann-json3-dev \
-    libbpf-dev linux-tools-generic python3 python3-pyqt5
+    libbpf-dev linux-tools-generic contrack-tools python3 python3-pyqt5
```

Comment:
I think this might have a typo https://packages.ubuntu.com/source/hirsute/conntrack-tools ?

Reply:
You're entirely correct! Sorry about that.

Have updated the PR accordingly